qcelemental/physical_constants/context.py

```diff
@@ -178,6 +179,7 @@ def Quantity(self, data: str) -> 'Quantity':
 
         return self.ureg.Quantity(data)
 
+    @lru_cache()
```

Comment:
Do you want to set an LRU cache `maxsize` here like was done with the reconcile_nucleus function?

Reply:
I guessed the 128 was fine. Total number of conversion pairs used is small.